src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,177 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String TIME_ZONE_EUROPE = "Europe/Paris";
+    private static final String DATE_FORMAT = "yyyyMMdd";
+    private static final String DATE_MONTH_REDUCTION_FORMAT = "d MMM yyyy";
+    private static final String DATE_TIME_FORMAT = "dd MMMM yyyy HH:mm";
```

Comment:
some of these formatters probably already exists  https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html

Reply:
Fixed (found replace only for 'DATE_FORMATE')